app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -255,16 +258,23 @@ private function getOrCreateUser($record, $values)
                 'source' => env('NORTHSTAR_CLIENT_ID'),
             ];
 
-            if ($record['Phone']) {
-                $userData['sms_status'] = $this->transformSmsStatus($record['Opt-in to Partner SMS/robocall']);
+            $recordEmailOptIn = $record['Opt-in to Partner email?'];
+            if ($recordEmailOptIn) {
+                $userData['email_subscription_status'] = str_to_boolean($recordEmailOptIn);
+            }
+
+            // Note: Not a typo -- this column name does not have the trailing question mark.
+            $recordSmsOptIn = $record['Opt-in to Partner SMS/robocall'];
+            if ($recordSmsOptIn && $recordMobile) {
```

Comment:
same here - do we need to check if `isset($recordSmsOptIn) && !empty($recordSmsOptIn)` here?